src/components/ModalOpen.tsx

```diff
@@ -168,6 +169,32 @@ class ModalOpenInternal extends React.Component<ModalOpenInternalProps, ModalOpe
     return file;
   }
 
+  // it is not guaranteed that the File System Access API is available on all
+  // browsers. If the function is not available, a fallback behavior is used.
+  onUpload = async (_: any, files: Result[]) => {
```

Comment:
[optional] Consider rnaming to `onFileChanged` to align with the event name.

Reply:
done